scripts/opx_build

```diff
@@ -53,6 +53,10 @@ repos=( opx-logging
 	opx-tmpctl
 	)
 
+if [ ! -z "$1" ] && [ "$1" != "all" ]; then
```

Comment:
Now I see that you have renamed the script to opx_build. Please ignore the comments about script name. But I think the "all" is still unnecessary.